src/SDK/Subscriber/Listener/StartSpanListener.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Subscriber\Listener;
+
+use function count;
+use OpenTelemetry\SDK\Attributes;
+use OpenTelemetry\SDK\AttributesInterface;
+use OpenTelemetry\SDK\Trace\Span;
+use OpenTelemetry\SDK\Subscriber\ListenerInterface;
+
+class StartSpanListener implements ListenerInterface
+{
+
+	public function action(Span $span):void
```

Comment:
A method name should be a verb.
A listener/subscriber should depend on events or callbacks not on concrete objects, otherwise it would be an observer.

Reply:
In the Otel-php module case isn't the observer mechanism suited better with the user registering the observer with the Events to be listened?